README.md

```diff
@@ -197,3 +197,7 @@ iOS13 and higher
 
 iOS12 and lower
 * NSBluetoothPeripheralUsageDescription
+
+#### How to add reactive ble lib to proguard rules (Android)
+
+In case you are using proguard add the following snippet to your `proguard-rules.pro` file: `-keep class com.signify.hue.** { *; }`. This will prevent issues like #131 .
```

Comment:
I suggest the followingn changes:

- fix the capitalization in "ProGuard"
- replace "snippet" with "rule"
- consider using a code block (instead of an in-line code span)
- replace `#131` with `[#131](https://github.com/PhilipsHue/flutter_reactive_ble/issues/131)` (it's not converted to a link automatically, at least I don't see it happening)